tests/fixtures/galaxy_fx.py

```diff
@@ -69,32 +65,32 @@ def mock_system_api_server_fx():
         )
         # - Prae Flyi RO-I b29-113
-        httpserver.expect_request("/search", query_string=b"name=Prae+Flyi+RO-I+b29-113&limit=1").respond_with_data(
-            """{"meta": {"name": "Prae Flyi RO-I b29-113", "type": "Perfect match"}, "data": [{"name": "Prae Flyi RO-I b29-113", "similarity": "Perfect match","id":22311274}]}"""
+        httpserver.expect_request("/api/systems", query_string=b"filter%5Bname:ilike%5D=Prae+Flyi+RO-I+b29-113&sort=name&limit=1").respond_with_data(
+            """{"data": [{"id": "249152528933625", "type": "systems", "attributes": {"name": "Prae Flyi RO-I b29-113", "coords": {"x": -586.125, "y": -112.0625, "z": 39248.5}}, "links": {"self": "http://sapi.fuelrats.dev/api/systems/249152528933625"}, "related": {}, "relationships": {"planets": {"data": [], "links": {"self": "http://sapi.fuelrats.dev/api/systems/249152528933625/relationships/planets", "related": "http://sapi.fuelrats.dev/api/systems/249152528933625/planets"}, "meta": {"direction": "ONETOMANY", "results": {"limit": 10, "available": 0, "returned": 0}}}, "stars": {"data": [], "links": {"self": "http://sapi.fuelrats.dev/api/systems/249152528933625/relationships/stars", "related": "http://sapi.fuelrats.dev/api/systems/249152528933625/stars"}, "meta": {"direction": "ONETOMANY", "results": {"limit": 10, "available": 0, "returned": 0}}}}, "meta": {}}], "included": [], "links": {"first": "http://sapi.fuelrats.dev/api/systems?sort=name&filter%5Bname%3Ailike%5D=Prae+Flyi+RO-I+b29-113&page%5Boffset%5D=0", "last": "http://sapi.fuelrats.dev/api/systems?sort=name&filter%5Bname%3Ailike%5D=Prae+Flyi+RO-I+b29-113&page%5Boffset%5D=0", "self": "http://sapi.fuelrats.dev/api/systems?filter[name:ilike]=Prae+Flyi+RO-I+b29-113&sort=name&limit=1"}, "meta": {"results": {"available": 1, "limit": 10, "offset": 0, "returned": 1}}}"""
         )
         # - Chua Eohn CT-F d12-2
-        httpserver.expect_request("/search", query_string=b"name=Chua+Eohn+CT-F+d12-2&limit=1").respond_with_data(
-            """{"meta": {"name": "Chua Eohn CT-F d12-2", "type": "Perfect match"}, "data": [{"name": "Chua Eohn CT-F d12-2", "similarity": "Perfect match", "id": 19626238}]}"""
+        httpserver.expect_request("/api/systems", query_string=b"filter%5Bname:ilike%5D=Chua+Eohn+CT-F+d12-2&sort=name&limit=1").respond_with_data(
```

Comment:
It is probably a good idea to properly encode URLs with quote_plus(), you should not expect endpoints to handle incorrectly encoded URL characters.

Reply:
Nice catch, I believe we [found this out the hard way in mecha2](https://github.com/FuelRats/pipsqueak/pull/317).

Reply:
I've updated the `_call` method to use `urllib.urlencode` rather than `html.escape`. I think, however, I'm going to leave the test data as-is. The point of the mock data is to represent a known constant value, and calculating URIs at runtime would be unnecessary for that purpose.